src/sagemaker_pytorch_serving_container/default_inference_handler.py

```diff
@@ -31,10 +43,18 @@ def default_model_fn(self, model_dir):
 
         Returns: A PyTorch model.
         """
-        raise NotImplementedError(textwrap.dedent("""
-        Please provide a model_fn implementation.
-        See documentation for model_fn at https://github.com/aws/sagemaker-python-sdk
-        """))
+        if os.getenv(INFERENCE_ACCELERATOR_PRESENT_ENV) == 'true':
+            default_model_filename = "model.pt"
+            model_path = os.path.join(model_dir, default_model_filename)
+            if not os.path.exists(model_path):
+                raise FileNotFoundError("Cannot find model.pt.")
```

Comment:
can we use a similar error message as: https://github.com/aws/sagemaker-mxnet-serving-container/blob/master/src/sagemaker_mxnet_serving_container/default_inference_handler.py#L52

Reply:
updated the error message and referenced to the default model name constant